### PR TITLE
translate sharding-scaling README.md document to chinese

### PR DESCRIPTION
**Pull request by strongduanmu**
Fixes [#4906 ](https://github.com/apache/incubator-shardingsphere/issues/4906)

Changes proposed in this pull request:
- translate sharding-scaling README.md document to chinese

### CODE REVIEW

File: sharding-scaling/README_ZH.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# ShardingScaling - ShardingSphere横向扩展组件
```
Comment (kimmking) [inline sharding-scaling/README_ZH.md]: 水平扩展  is better

File: sharding-scaling/README_ZH.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+关于图片的补充说明：
+
+1. 只支持迁移整个数据库，暂不支持迁移指定的表。
+2. 迁移过程分为两步——历史数据迁移和实时数据迁移。
+  - 在历史数据迁移过程中，Sharding-Scaling使用 `select *`语句去获取数据，使用`insert`语句将数据迁移到目标库中；
+  - 在实时数据迁移过程中，Sharding-Scaling使用binlog来迁移数据，并且在迁移前会标记下binlog位置。
+
+3. 如果源schema中的表有主键，Sharding-Scaling就能够使用 `where`声明的条件，并发地迁移表中的数据。
+
+## 必备条件
```
Comment (kimmking) [inline sharding-scaling/README_ZH.md]: 环境准备 is better

File: sharding-scaling/README_ZH.md (hunk 1 above)
Comment (KomachiSion) [reply sharding-scaling/README_ZH.md]: it should be `ShardingSphere弹性伸缩组件`

File: sharding-scaling/README_ZH.md (hunk 2 above)
Comment (KomachiSion) [reply sharding-scaling/README_ZH.md]: I also think `环境准备` is better.

Comment (strongduanmu) [reply sharding-scaling/README_ZH.md]: Thanks a lot, I'll modify it.

File: sharding-scaling/README_ZH.md (hunk 1 above)
Comment (strongduanmu) [reply sharding-scaling/README_ZH.md]: 弹性伸缩组件 is a goods translation, i will modify it.

**Pull request merged**